docs/docsite/rst/installation_guide/intro_installation.rst

```diff
@@ -5,531 +5,129 @@
 
-Installing Ansible with ``pip``
--------------------------------
-
-.. note::
-
-	If you have Ansible 2.9 or older installed or Ansible 3, see :ref:`pip_upgrade`.
+.. _pip_install:
 
-Once ``pip`` is installed, you can install Ansible::
-
-    $ python -m pip install --user ansible
-
-In order to use the ``paramiko`` connection plugin or modules that require ``paramiko``, install the required module [1]_::
-
-    $ python -m pip install --user paramiko
-
-If you wish to install Ansible globally, run the following commands::
-
-    $ sudo python get-pip.py
-    $ sudo python -m pip install ansible
-
-.. note::
-
-    Running ``pip`` with ``sudo`` will make global changes to the system. Since ``pip`` does not coordinate with system package managers, it could make changes to your system that leaves it in an inconsistent or non-functioning state. This is particularly true for macOS. Installing with ``--user`` is recommended unless you understand fully the implications of modifying global files on the system.
-
-.. note::
-
-    Older versions of ``pip`` default to http://pypi.python.org/simple, which no longer works.
-    Please make sure you have the latest version of ``pip`` before installing Ansible.
-    If you have an older version of ``pip`` installed, you can upgrade by following `pip's upgrade instructions <https://pip.pypa.io/en/stable/installing/#upgrading-pip>`_ .
-
-.. _from_pip_venv:
-
-Installing Ansible in a virtual environment with ``pip``
---------------------------------------------------------
-
-.. note::
-
-  If you have Ansible 2.9 or older installed or Ansible 3, see :ref:`pip_upgrade`.
-
-Ansible can also be installed inside a new or existing ``virtualenv``::
-
-    $ python -m virtualenv ansible  # Create a virtualenv if one does not already exist
-    $ source ansible/bin/activate   # Activate the virtual environment
-    $ python -m pip install ansible
-
-.. _pip_upgrade:
-
-Upgrading Ansible with ``pip``
-------------------------------
-
-Upgrading from 2.9 or earlier to 2.10
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Installing Ansible
+------------------
 
-Starting in version 2.10, Ansible is made of two packages. When you upgrade from version 2.9 and older to version 2.10 or later, you need to uninstall the old Ansible version (2.9 or earlier) before upgrading. If you do not uninstall the older version of Ansible, you will see the following message, and no change will be performed:
+Use ``pip`` in your selected Python environment to install the Ansible package of your choice for the current user:
 
 .. code-block:: console
 
-    Cannot install ansible-base with a pre-existing ansible==2.x installation.
-
-    Installing ansible-base with ansible-2.9 or older currently installed with
-    pip is known to cause problems. Please uninstall ansible and install the new
-    version:
+    $ python3 -m pip install --user ansible
```

Comment:
I don't think we need to go into a lot of details but should there be a brief mention of python virtual environments and how it compares to ``--user`` ? We can certainly link to https://docs.python.org/3/tutorial/venv.html for more information.

Reply:
I specifically removed venv instructions.  If users want to use a venv, they should already know what it is